css/style.css

```diff
@@ -415,6 +454,29 @@ a {
   align-items: center;
 }
 
+.banner-color {
+  background: linear-gradient(to bottom, rgba(235, 235, 250, 1) 0%, rgba(190, 192, 234, 1) 100%);
+  padding-top: 100px;
+}
+
 .column {
   flex: 1;
 }
+
+.color-class {
+  color: #f7f7f7;
+}
+
+.button-link {
+  text-decoration: none;
+  background-color:#f7f7f7;
+  display: inline-block;
+  border-radius: 50%;
```

Comment:
Try to use px in border-radius since in some browsers could present some bugs, here's good so don't change it